gcloud-java-compute/src/test/java/com/google/gcloud/compute/it/ITComputeTest.java

```diff
@@ -855,4 +875,464 @@ public void testListGlobalAddresses() throws InterruptedException {
     compute.delete(firstAddressId);
     compute.delete(secondAddressId);
   }
+
+  @Test
+  public void testCreateGetAndDeleteStandardDisk() throws InterruptedException {
+    String name = BASE_RESOURCE_NAME + "create-and-get-standard-disk";
+    DiskId diskId = DiskId.of(ZONE, name);
+    DiskInfo diskInfo =
+        DiskInfo.of(diskId, StandardDiskConfiguration.of(DiskTypeId.of(ZONE, "pd-ssd"), 100L));
+    Operation operation = compute.create(diskInfo);
+    while (!operation.isDone()) {
```

Comment:
Yes the operation is done when the disk is READY and yes it is returned in CREATING status.